Cargo.toml

```diff
@@ -42,13 +42,15 @@ hyper = { version = "^0.14.10", features = [ "server", "tcp", "http1", "http2" ]
 impl-serde = "0.3.2"
 itertools = "0.10"
 mimalloc = { version = "0.1", default-features = false, optional = true }
+num-bigint = { version = "0.4", default-features = false, features = ["rand"] }
 num_cpus = "1.13"
-once_cell = "1.9"
+once_cell = "1.10"
 prometheus = { version = "0.13", features = [ "process" ] }
 proptest = { version = "1.0", optional = true }
 reqwest = "0.11.8"
 serde = "1.0"
 serde_json = "1.0"
+semaphore = { git = "https://github.com/worldcoin/semaphore-rs" }
```

Comment:
When using a git repo as dependency it's good practice to specify a specific commit hash